### PR TITLE
Azure configuration

### PR DESCRIPTION
**Pull request by jjh121**
- Add serilog's app insights sink
- Add appsettings.Test.json in order to have Azure use this configuration
- Updated nuget packages

### CODE REVIEW

File: FoxHound.Web/appsettings.Test.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (jjh121) [inline FoxHound.Web/appsettings.Test.json:1]: I didn't know is this environment for Azure should be considered Test...but that's what i went with

**Comment by tnavarra** [comment]
I didn't see the client telemetry included anywhere in this.  Were you planning on adding that in a future PR?  I don't mind adding it.

File: FoxHound.Web/appsettings.Test.json (hunk 1 above)
Comment (tnavarra) [reply FoxHound.Web/appsettings.Test.json:1]: Makes sense.  My guess is the plan is to configure the instrumentation key via the configuration in the app service.  Consider an empty placeholder in the root appsettings for this just so its obvious what configuration exists.

Comment (jjh121) [reply FoxHound.Web/appsettings.Test.json:1]: Yes, i will configure the key in configuration in the app service. Good idea about adding placeholder...it's been added.

**Comment by jjh121** [comment]
> I didn't see the client telemetry included anywhere in this. Were you planning on adding that in a future PR? I don't mind adding it and submitting a PR.

I was thinking about how the client would work and doing that as a separate PR. I don't mind if you take that one. I'm interested in how we get the app insights key into the mix. Maybe it lives in the .env file? Should that key be considered sensitive?

**Comment by tnavarra** [comment]
> > I didn't see the client telemetry included anywhere in this. Were you planning on adding that in a future PR? I don't mind adding it and submitting a PR.
> 
> I was thinking about how the client would work and doing that as a separate PR. I don't mind if you take that one. I'm interested in how we get the app insights key into the mix. Maybe it lives in the .env file? Should that key be considered sensitive?

I don't mind taking it.  It will give me a chance to contribute here as well.  By its nature I think there is no built in way around having that key exposed on the client (if you want to do client side analytics).  Some interesting links [here](https://stackoverflow.com/questions/54535275/what-will-happen-if-applicationinsights-instrumentationkey-gets-stolen).

**Comment by jjh121** [comment]
> > > I didn't see the client telemetry included anywhere in this. Were you planning on adding that in a future PR? I don't mind adding it and submitting a PR.
> > 
> > 
> > I was thinking about how the client would work and doing that as a separate PR. I don't mind if you take that one. I'm interested in how we get the app insights key into the mix. Maybe it lives in the .env file? Should that key be considered sensitive?
> 
> I don't mind taking it. It will give me a chance to contribute here as well. By its nature I think there is no built in way around having that key exposed on the client (if you want to do client side analytics). Some interesting links [here](https://stackoverflow.com/questions/54535275/what-will-happen-if-applicationinsights-instrumentationkey-gets-stolen).

That is interesting, I think for now it's fine to just have it in the .js code.

**Pull request merged**